strawberry/field.py

```diff
@@ -343,7 +368,7 @@ def _has_async_base_resolver(self) -> bool:
 
     @cached_property
     def is_async(self) -> bool:
-        return self._has_async_permission_classes or self._has_async_base_resolver
+        return self._has_async_base_resolver
```

Comment:
Don't we need to check if we have async extensions as well?

Reply:
No, since that is handled by the `SchemaConverter`. The field itself does not access the extensions, only the schema converters and extension creators access the field. An async extension might make a wrapped field resolver async, but since it's handled by the extension, this doesn't change the `is_async` property of the field.